front/app/components/UI/Button/index.tsx

```diff
@@ -11,7 +11,7 @@ import { RouteType } from 'routes';
 import Link from 'utils/cl-router/Link';
 
 interface Props extends ButtonProps {
-  linkTo?: RouteType | null;
+  linkTo?: string | null;
```

Comment:
This component can also link to external links so the typing had to change to the more generic `string`.